### PR TITLE
Update resources_common_properties.md

### PR DESCRIPTION
**Pull request by mbaitelman**
Add :quiet 
See https://github.com/chef/chef/blob/master/lib/chef/resource.rb#L540

### Description

Documents missing option 
### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: layouts/shortcodes/resources_common_properties.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ The following properties are common to every resource:
 
 `ignore_failure`
 
-:   **Ruby Type:** true, false \| **Default Value:** `false`
+:   **Ruby Type:** true, false, :quiet \| **Default Value:** `false`
 
     Continue running a recipe if a resource fails for any reason.
```
Comment (tas50) [inline layouts/shortcodes/resources_common_properties.md:7]: We should document what quiet actually does here

File: layouts/shortcodes/resources_common_properties.md [hunk 2]
```diff
@@ -2,10 +2,11 @@ The following properties are common to every resource:
...
-
+    Using `:quiet` logs the full error though will still continue running the recipe.
```
Comment (IanMadd) [inline layouts/shortcodes/resources_common_properties.md:8]:
```suggestion

    Using `:quiet` logs the full error and the recipe will continue to run if a resource fails.
```

**Review by IanMadd — COMMENTED** [review]
@mbaitelman Could you add your DCO signoff.
https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco

File: layouts/shortcodes/resources_common_properties.md (hunk 2 above)
Comment (tas50) [reply layouts/shortcodes/resources_common_properties.md:8]: I'm not sure if that's the correct description. Using :quiet prevents a full error stacktrace from being shown.

Comment (IanMadd) [reply layouts/shortcodes/resources_common_properties.md:8]: So maybe

`:quiet` will not display the full stack trace and the recipe will continue to run if a resource fails.

**Comment by mbaitelman** [comment]
Created #2541 to replace this PR. 
It includes the DCO and requested updates.

**Pull request closed without merging**